R/functions_deprecated.R

```diff
@@ -1,8 +1,8 @@
-######################################
+#################################### #
 #
 # global utility functions for the TSS project
 #
-######################################
+#################################### #
```

Comment:
If really deprecated but could still be used accidentally, then [`.Deprecated`](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/Deprecated). Otherwise just delete (after all, we still keep them in past commits if we need to recover them).

Reply:
Solved in last commit!

Reply:
I arrive a bit after the fight, but I can't remember using these functions ever (so at least they are not needed on my branch).

Reply:
Can’t we just delete this whole script then?

Reply:
Done!